docs/iris/src/whatsnew/1.9.rst

```diff
@@ -0,0 +1,13 @@
+Iris 1.9 Bugs Fixed
+===================
+Iris 1.9 Features
+=================
+Iris 1.9 Incompatible Changes
+=============================
+Iris 1.9 Deprecations
+=====================
+Iris 1.9 Documentation Changes
+==============================
+* Added a first pass of an aggregator to build a "What's New" document from submitted RST files for each change.
+ * Added a new page to the dev docs to explain the format for an entry.
+ * Added a reference to the new page, to the Dev Docs index.
```

Comment:
> I suggest that this file is not under source control
That was the intention, I simply forgot to delete it - my fault, I did the same with 1.8 (which broke the merge.)

In theory, making edits to the contributions would be the best way, but I can see it as possible that a need to add an extra message to a particular release may be possible - for which having the 'final' version in source control (as currently) allows. A 'finished' version won't be overwritten.